### PR TITLE
Implemented a barcode and qrcode scanner(1uvpmn4)

### PR DESCRIPTION
**Pull request by disha1202**
Screenshot from 2021-11-24 14-49-09
Screenshot from 2021-11-24 14-53-29

### CODE REVIEW

File: src/views/Scanner.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { StreamBarcodeReader } from "vue-barcode-reader";
+import {modalController } from '@ionic/vue';
+export default {
+    name: 'Scanner',
+    components:{
+        StreamBarcodeReader,
+    },
+    methods: {
+    onDecode (result) {
+      console.log(result);
```
Comment (ymaheshwari1) [inline src/views/Scanner.vue:20]: Remove console statements

File: src/views/Scanner.vue [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="scanner">
```
Comment (ymaheshwari1) [inline src/views/Scanner.vue:2]: Use consistent indentation

File: src/views/BarcodeSample.vue [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<template>
+    <vue-barcode  value="123456734567"></vue-barcode>
```
Comment (ymaheshwari1) [inline src/views/BarcodeSample.vue:2]: Use self-closing tag

File: src/views/Scanner.vue [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+<script>
+import { StreamBarcodeReader } from "vue-barcode-reader";
+import {modalController } from '@ionic/vue';
+export default {
+    name: 'Scanner',
+    components:{
+        StreamBarcodeReader,
+    },
+    methods: {
+    onDecode (result) {
```
Comment (ymaheshwari1) [inline src/views/Scanner.vue:19]: Improve indentation

File: src/views/Scanner.vue (hunk 1 above)
Comment (disha1202) [reply src/views/Scanner.vue]: Removed

File: src/views/Scanner.vue (hunk 2 above)
Comment (disha1202) [reply src/views/Scanner.vue]: Done

File: src/views/BarcodeSample.vue (hunk 3 above)
Comment (disha1202) [reply src/views/BarcodeSample.vue]: Done

File: src/views/Scanner.vue (hunk 4 above)
Comment (disha1202) [reply src/views/Scanner.vue]: Improved indentation

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]
Remove an extra space.

**Comment by disha1202** [comment]
> Remove an extra space. https://github.com/hotwax/ionic-sdk/pull/2/files#diff-c338b8b3faa4fd2eda1f87c1b17f209ddc158730c477b591f0f250d9974c95e4R2

Done sir

**Pull request closed without merging**